Makefile

```diff
@@ -20,14 +20,14 @@ create : delete build
 
 	kubectl wait node --for condition=ready --all --timeout=60s
 
-	# Install prometheus
-	#@kubectl apply -f ${ISTIO_HOME}/samples/addons/prometheus.yaml
```

Comment:
can you double check this?  Makefile is really particular about tabs vs. spaces and it looks like you replaced a tab with 4 spaces.

Reply:
I only did this to remove printing of commented shell commands. TAB (or shell recipes) get printed (if not preceded with @).

Should I re-replace them with tabs?